README.md

```diff
@@ -13,27 +14,28 @@ meant to be an exercise to help us gauge where you are in terms of your current
 and be a talking-point during the interview. A poor analysis will not necessarily
 disqualify you from the job.
 
-### Your Task:
-Use Python to perform a simple analysis of the data table from the following url:
-https://firms.modaps.eosdis.nasa.gov/active_fire/viirs/shapes/zips/VNP14IMGTDL_NRT_Global_7d.zip
-We would like you to analyse the data in relation to the areas of interest in the ares.geojson file.
-At minimum we would like to see at least one map and statistical chart.
+### The data
+Our main focus right now are supply chains, and how they affect the big challenges we face: climate change, water&food security and biodiversity lost. 
 
-### Hints and resources
+We would like you to use one of the datasets we are familiar with: [Mapspam](https://www.mapspam.info/data/)
```

Comment:
I would give all of them the same dataset, so we can have a common starting point (dataset +areas)

Reply:
Yes Mapspam is a huge dataset and for me is the same id they analyse soybean that wheat, as I say the data is similar if not the same and it gives them room enough to be creative.

Reply:
Also it is a rest to see if they understand the variables and the metadata

Reply:
I've been looking at the data and for example there is barely any wheat in the areas of interest we are attaching. I'll propose to use soy and yams